common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1440,6 +1450,55 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(8)
 
+  val PARTITION_SPLIT_THRESHOLD: ConfigEntry[Long] =
+    buildConf("celeborn.shuffle.partitionSplit.threshold")
+      .withAlternative("rss.partition.split.threshold")
+      .categories("client")
+      .doc("Shuffle file size threshold, if file size exceeds this, trigger split.")
+      .version("0.2.0")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("256m")
```

Comment:
WDYT if we change the default value to 1G?

Reply:
necessary